### PR TITLE
reorganized basics section

### PR DESCRIPTION
**Pull request by RiccardoGrigoletto**

### CODE REVIEW

**Review by AAnastasiia11 — APPROVED** [review]

File: notebooks/featurestore/hsfs/basics/feature_exploration.ipynb [hunk 1]
```diff
@@ -1,676 +0,0 @@
-{
```
Comment (SirOibaf) [inline notebooks/featurestore/hsfs/basics/feature_exploration.ipynb:1]: why did you remove this notebook?

**Review by SirOibaf — CHANGES_REQUESTED** [review]

File: notebooks/featurestore/hsfs/basics/feature_exploration.ipynb (hunk 1 above)
Comment (RiccardoGrigoletto) [reply notebooks/featurestore/hsfs/basics/feature_exploration.ipynb:1]: @AAnastasiia11  are you sure you were asked to get feature_exploration to be deleted?

Comment (RiccardoGrigoletto) [reply notebooks/featurestore/hsfs/basics/feature_exploration.ipynb:1]: @SirOibaf Ok we figured it out, it has to stay there. @AAnastasiia11 will change more stuff in this card so I'll close this PR and she will open a new one.
